langchain/document_loaders/nuclia.py

```diff
@@ -0,0 +1,35 @@
+"""Loader that loads Nuclia Understanding API results."""
+import json
+import uuid
+from typing import List
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
+from langchain.tools.nuclia.tool import NucliaUnderstandingAPI
+
+
+class NucliaLoader(BaseLoader):
+    """Loader that loads Nuclia Understanding API results."""
+
+    def __init__(self, path: str, nuclia_tool: NucliaUnderstandingAPI):
+        self.nua = nuclia_tool
+        self.id = str(uuid.uuid4())
+        self.nua.run(
```

Comment:
would it be cleaner to store path here and then run this command in L23 with self.path?

Reply:
Actually we need the `path` value only here (line 18).
On line 23, we do `"path": None`.